erizo_controller/common/semanticSdp/SDPInfo.js

```diff
@@ -430,12 +430,6 @@ class SDPInfo {
 
       sdp.media.push(md);
     });
-    bundle.mids.sort();
```

Comment:
We added these lines for Firefox, can you test that nothing is broken in Firefox with your PR?

Reply:
now the medias are ordered in cpp in the way the browser creates the offer. I'll double check but I've done all the tests